http/src/request/channel/message/create_message.rs

```diff
@@ -146,8 +144,7 @@ impl<'a> CreateMessage<'a> {
     /// Returns [`CreateMessageError::EmbedTooLarge`] if the embed is too large.
     ///
     /// [the discord docs]: https://discord.com/developers/docs/resources/channel#embed-limits
-    /// [`EmbedBuilder`]: ../../../../../twilight_embed_builder/builder/struct.EmbedBuilder.html
-    /// [`CreateMessageError::EmbedTooLarge`]: enum.CreateMessageError.html#variant.EmbedTooLarge
+    /// [`EmbedBuilder`]: https://docs.rs/twilight-embed-builder/*/twilight_embed_builder
```

Comment:
Is there a way to make this link to paths on `api.twilight.rs` and `docs.rs`?

Reply:
yes, keep the relative html link or add the project as dev dependency and use `::twilight_embed_builder...`

Reply:
I think adding the dev dep would be more future-proof

Reply:
hmm, i mixed it up with some other issues in the past. i tested it and rustdoc doesn't like it at all and the error message is kinda bugged.

<details>
<summary>stable</summary>

```
/// [`EmbedBuilder`]: ::twilight_embed_builder::EmbedBuilder
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ the module `create_message` contains no item named ``
```
</details>

<details>
<summary>nightly</summary>

```
/// [`EmbedBuilder`]: ::twilight_embed_builder::EmbedBuilder
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ no item named `` in scope
```
</details>